File: scala-spring/transactions-command-side-web/src/main/scala/net/chrisrichardson/eventstore/examples/bank/web/transactions/CommandSideWebTransactionsConfiguration.scala
package net.chrisrichardson.eventstore.examples.bank.web.transactions

import net.chrisrichardson.eventstore.examples.bank.transactions.TransactionConfiguration
import org.springframework.context.annotation.{ComponentScan, Configuration, Import}

@Configuration
@Import(Array(classOf[TransactionConfiguration]))
@ComponentScan
class CommandSideWebTransactionsConfiguration {

}

File: java-spring/transactions-command-side-web/src/main/java/net/chrisrichardson/eventstore/javaexamples/banking/web/commandside/transactions/CommandSideWebTransactionsConfiguration.java
package net.chrisrichardson.eventstore.javaexamples.banking.web.commandside.transactions;

import net.chrisrichardson.eventstore.javaexamples.banking.backend.commandside.transactions.MoneyTransferConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;

@Configuration
@Import({MoneyTransferConfiguration.class})
@ComponentScan
public class CommandSideWebTransactionsConfiguration  {

}
